web/src/app.tsx

```diff
@@ -52,17 +68,41 @@ export class App extends React.Component<{}, AppState> {
   };
 
   private getPageContent = (id: string) => {
+    console.debug('getPageContent', id);
     switch (id) {
-      case 'netflow-traffic-parent':
+      case 'pod-tab':
```

Comment:
what is the rationale for having here only pod/namespace/node and not all the supported kinds? Is it only used in tests?

Reply:
yes it's just for local dev purposes and it's hardcoding "test" as name and "default" as namespace